store/server/features/dashboard/employee-status/queries.ts

```
@@ -1,8 +1,8 @@
+import { requestHeader } from '@/helpers/requestHeader';
 import { useAuthenticationStore } from '@/store/uistate/features/authentication';
```

ESLint

[error] 'useAuthenticationStore' is defined but never used.

(@typescript-eslint/no-unused-vars)